
package it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.3.2
 * 2020-12-21T09:41:23.713+01:00
 * Generated source version: 3.3.2
 */

@WebFault(name = "fault3", targetNamespace = "http://pep.base.nmsf.csi.it/")
public class AuthException_Exception extends Exception {

    private it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep.AuthException fault3;

    public AuthException_Exception() {
        super();
    }

    public AuthException_Exception(String message) {
        super(message);
    }

    public AuthException_Exception(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public AuthException_Exception(String message, it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep.AuthException fault3) {
        super(message);
        this.fault3 = fault3;
    }

    public AuthException_Exception(String message, it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep.AuthException fault3, java.lang.Throwable cause) {
        super(message, cause);
        this.fault3 = fault3;
    }

    public it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep.AuthException getFaultInfo() {
        return this.fault3;
    }
}
